code/menuui/playermenu.cpp

```diff
@@ -1391,7 +1392,17 @@ void player_tips_init()
 			if (Num_player_tips >= MAX_PLAYER_TIPS) {
 				break;
 			}
-			Player_tips[Num_player_tips++] = stuff_and_malloc_string(F_NAME, NULL);
+
+			//SCP_string str =  message_translate_tokens(stuff_and_malloc_string(F_NAME, NULL));  
+			//char* tmp = &str[0];
+			//Player_tips[Num_player_tips++] = tmp;
+		    //std:: vector<char> v(str.begin(), str.end());
+			// char* ca = &v[0]; // pointer to start of vector
+		    //char* chr = const_cast<char*>(str.c_str());
+			// Player_tips[Num_player_tips++] = chr; 
```

Comment:
While it is normal for people experimenting with a solution to write code that is later found to be unnecessary, we generally do not expect that code to be committed to the main code repository. Code that is commented out should not be part of your PR, unless there are specific reasons to include those lines, which should be laid out in a comment.

Reply:
I'd like to mention here that Github has the option of making PR Drafts, which show that the contribution is still a work in progress.  Standard PR's are assumed to be work that is completed, functioning on the author's machine, and ready for review.

Reply:
Thank you guys!